packages/base-contract/src/index.ts

```diff
@@ -146,6 +150,9 @@ export class BaseContract {
         }
         return rawEncoded;
     }
+    public getWeb3Wrapper(): Web3Wrapper {
```

Comment:
This exposes `web3-wrapper` on all contract public interfaces. The fact that we use `web3-wrapper` is an implementation detail and we don't want to surfaced to contract users. Why was this added?

Reply:
This was added for unit tests. Simplifying the tests and getting rid of it.

Reply:
@xianny within tests, it's also kosher to do `(baseContract as any)._web3Wrapper`